rgwadmin/user.py

```
@@ -187,16 +194,19 @@ def diff(self):
     def save(self):
         rgw = RGWAdmin.get_connection()
         try:
-            existing = rgw.get_metadata('user', self.user_id)
+            self.fetch(self.user_id)
```

Won't this never raise NoSuchKey since fetch() swallows that exception?